src/sections/Pricing/comparison.js

```diff
@@ -132,15 +133,20 @@ h2, h5{
   margin-top: 1rem;
 }
 .details{
+  display: flex;
+  margin: 0.5rem 0;
```

Comment:
This needs to be applied to the `.pricing-feature features` CSS in the `pricing.style.js` file. Not here.

Reply:
Ok sorry for misunderstanding